### DISCUSSION TITLE
is it possible for extension 'mentions' or 'suggestion' to have multiple denotation characters?

### DISCUSSION THREAD

**Discussion by ElHex** [ueberdosis/tiptap#1949] [General Questions]
hey!

by default it only uses @, but can it use it besides another character? for example:

1. @
2. / 

etc

**Comment by philippkuehn** [comment]
You can re-use the [`@tiptap/suggestion`](https://tiptap.dev/api/utilities/suggestion) package.

See the `@tiptap/extension-mention` package or the [commands experiment](https://tiptap.dev/experiments/commands) as a starting point.
